barretenberg/cpp/src/barretenberg/commitment_schemes/zeromorph/zeromorph.hpp

```diff
@@ -69,9 +71,6 @@ template <typename Curve> class ZeroMorphProver_ {
                                                                  std::span<const FF> u_challenge)
     {
         size_t log_N = numeric::get_msb(polynomial.size());
-        // The size of the multilinear challenge must equal the log of the polynomial size
-        ASSERT(log_N == u_challenge.size());
```

Comment:
is this still true?

Reply:
I think you're mistakenly commenting on the deleted line?

Reply:
no, I was wondering if we should've kept this deleted line?